docs/source/transactions.md

```diff
@@ -709,6 +710,140 @@ Command to change Pool's configuration
 }
 ```
 
+#### AUTH_RULE
+
+Transaction to change authentication rules. 
+Authentication rules are stored in the State as key - value dictionary. This coincides with the storage structure in (auth_map)[auth_rule.md]. 
+If config ledger does not have transaction for key, the State use value from auth_map.
```

Comment:
Better to say:
`If config ledger does not have a transaction for a given key, then a default rule defined in code is used.`